crates/kornia-dnn/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "kornia-dnn"
+authors.workspace = true
+categories.workspace = true
+description.workspace = true
+edition.workspace = true
+homepage.workspace = true
+include.workspace = true
+license.workspace = true
+license-file.workspace = true
+readme.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+version.workspace = true
+
+[dependencies]
+kornia-core = { workspace = true }
+kornia-image = { workspace = true }
+ort = { version = "2.0.0-rc.4", features = [
+    "load-dynamic",
```

Comment:
I'm generally against libraries setting features that affect linking like `load-dynamic` because Cargo features are additive workspace-wide; if an application's dependency sets `load-dynamic` for ort, there's no way for the developer to unset it. If a user of `kornia-dnn` wants to ship with CPU-only ort they'd be forced to compile their own dynamic library (since only CUDA doesn't ship with static libraries).

While I'd imagine most users *would* be using CUDA and thus probably the `load-dynamic` feature, I think it would be better to split this out into a passthrough feature like
```toml
[features]
ort-load-dynamic = [ "ort/load-dynamic" ]
ort-cuda = [ "ort/cuda" ]
```

and document accordingly, with a link to https://ort.pyke.io/setup/linking for more info. This way, end users are granted flexibility, but it isn't too much of a hassle for the majority of users looking to use the most common configuration.

Reply:
done